data-workflows/activity/tests/test_github_activity_model.py

```diff
@@ -1,30 +1,26 @@
-import re
-from datetime import datetime, timezone
+from datetime import datetime, timezone, time
 from unittest.mock import Mock
 
 import pytest
 from dateutil.relativedelta import relativedelta
 
 import activity
 from activity.github_activity_model import (
-    GitHubActivityType, transform_and_write_to_dynamo
+    transform_and_write_to_dynamo,
+    GitHubActivityType,
 )
 
 REPO1 = "demo/FOO"
 REPO2 = "org2/bar"
 PLUGIN_BY_REPO = {REPO1: "foo", REPO2: "bar"}
-PLUGIN_BY_EARLIEST_TS = {
-    REPO1: datetime.strptime("05/16/2023 10:24:20", "%m/%d/%Y %H:%M:%S"),
-    REPO2: datetime.strptime("06/26/2023 20:30:00", "%m/%d/%Y %H:%M:%S"),
-}
-FORMATTED_PLUGIN_BY_TS = {
-    repo: ts.replace(day=1) for repo, ts in PLUGIN_BY_EARLIEST_TS.items()
-}
 
 
-def generate_expected(data, granularity, type_id, ts_formatter):
+def generate_expected(data, granularity, type_id, ts_formatter, include_expiry=False):
```

Comment:
```suggestion
def generate_expected(data, granularity, type_id, ts_formatter, *, include_expiry=False):
```
nitpick (super-nit as this is in test code): I like boolean args to be keyword-only for clarity at the call site